session/mock_bootstrap.go

```diff
@@ -140,13 +155,20 @@ func modifyBootstrapVersionForTest(store kv.Storage, ver int64) int64 {
 	return ver
 }
 
+// MockUpgradeToVerLatestKind is used to indicate the use of different mock bootstrapVersion.
+var MockUpgradeToVerLatestKind = 1
```

Comment:
Split it to two vars like `mockUpgradeToVerLatest` and `mockSimpleUpgradeToVerLatest` is better